packages/wallets/src/features/cashier/modules/Transfer/components/TransferMessages/TransferMessagesConfig.tsx

```diff
@@ -0,0 +1,162 @@
+                values={{ formattedSourceCurrencyLimit }}
+            />
+        ) : (
+            <Trans
+                defaults='The daily transfer limit between your {{sourceAccountName}} and {{targetAccountName}} is {{formattedSourceCurrencyLimit}}.'
+                values={{
+                    formattedSourceCurrencyLimit,
+                    sourceAccountName,
+                    targetAccountName,
+                }}
+            />
+        ),
+    CUMULATIVE_TRANSFER_LIMIT_ALLOWED_DEMO: ({ formattedDemoLimit }: TMessage['values']) => (
+        <Trans
+            defaults='Your daily transfer limit for virtual funds is {{formattedDemoLimit}}'
+            values={{ formattedDemoLimit }}
+        />
+    ),
+    CUMULATIVE_TRANSFER_LIMIT_AVAILABLE: ({
+        formattedSourceCurrencyRemainder,
+        isTransferBetweenWallets,
+        sourceAccountName,
+        targetAccountName,
+    }: TMessage['values']) =>
+        isTransferBetweenWallets ? (
+            <Trans
+                defaults='The remaining daily transfer limit between your Wallets is {{formattedSourceCurrencyRemainder}}.'
+                values={{ formattedSourceCurrencyRemainder }}
+            />
+        ) : (
+            <Trans
+                defaults='The remaining daily transfer limit between your {{sourceAccountName}} and {{targetAccountName}} is {{formattedSourceCurrencyRemainder}}.'
+                values={{
+                    formattedSourceCurrencyRemainder,
+                    sourceAccountName,
+                    targetAccountName,
+                }}
+            />
+        ),
+    CUMULATIVE_TRANSFER_LIMIT_AVAILABLE_DEMO: ({ formattedDemoLimit }: TMessage['values']) => (
+        <Trans
+            defaults='Your remaining daily transfer limit for virtual funds is {{formattedDemoLimit}}.'
+            values={{ formattedDemoLimit }}
+        />
+    ),
+    CUMULATIVE_TRANSFER_LIMIT_REACHED: ({
+        formattedSourceCurrencyLimit,
+        isTransferBetweenWallets,
+        sourceAccountName,
+        targetAccountName,
+    }: TMessage['values']) =>
+        isTransferBetweenWallets ? (
+            <Trans
+                defaults='You have reached your daily transfer limit of {{formattedSourceCurrencyLimit}} between your Wallets. The limit will reset at 00:00 GMT.'
+                values={{ formattedSourceCurrencyLimit }}
+            />
+        ) : (
+            <Trans
+                defaults='You have reached your daily transfer limit of {{formattedSourceCurrencyLimit}} between your {{sourceAccountName}} and {{targetAccountName}}. The limit will reset at 00:00 GMT.'
+                values={{
+                    formattedSourceCurrencyLimit,
+                    sourceAccountName,
+                    targetAccountName,
+                }}
+            />
+        ),
+    LIFETIME_TRANSFER_LIMIT_ALLOWED_CRYPTO: ({ formattedSourceCurrencyLimit }: TMessage['values']) => (
```

Comment:
what about:

   LIFETIME_TRANSFER_LIMIT_ALLOWED_CRYPTO:  {
       text: "You have reached your daily transfer limit of {{formattedSourceCurrencyLimit}} between your Wallets. The limit will reset at 00:00 GMT.'"
       values: {
          formattedSourceCurrencyLimit,
          ... /* any other values */
        }
   }
   
   and then in specific component:
   <Trans
     default={msg.text}
     value={msg.values}
   >
   
   
  so the config would be cleaner

Reply:
@wojciech-deriv i have refactored the code
i removed TransferMessagesConfig and handle all messages inside validation functions